README.rst

```diff
@@ -245,7 +248,22 @@ make32_install_args
   macro in the ``.spec`` for the 32bit build. Again it is appended after
   make_install_args so 32bit specific overrides can be added.
 
-make_install_append
+prep_prepend
+  Additional actions that should take place directly after ``%prep``
+  This will be placed in the resulting ``.spec``, and is used for
+  situations where fine-grained control is required.
+
+build_prepend
+  Additional actions that should take place directly after ``%build``
+  This will be placed in the resulting ``.spec``, and is used for
+  situations where fine-grained control is required.
+
+install_prepend
+  Additional actions that should take place directly after ``%install``
+  This will be placed in the resulting ``.spec``, and is used for
+  situations where fine-grained control is required.
+
+install_append
   Additional actions that should take place after the ``make install`` step has
```

Comment:
I like the docstring better for this one, since it doesn't mention `make install`.

```
write out any custom supplied commands at the very end of the %install section
```

Can you update the README to describe the config file similarly?

Reply:
Wording updated.